### PR TITLE
Placeholder values to unblock testing

### PR DESCRIPTION
**Pull request by cmgiven**
A couple of empty arrays and a fix (?).

### CODE REVIEW

File: app/models/appeal.rb [hunk 1]
```diff
@@ -410,8 +410,8 @@ def fetch_post_decision_status
       :bva_decision
     elsif withdrawn?
       :withdrawn
-    else decision_issues.empty?
-         :other_close
+    else
+      :other_close
```
Comment (cmgiven) [inline app/models/appeal.rb]: I think the previous was a typo?

Comment (mc-usds) [reply app/models/appeal.rb]: D'oh it was. Thanks

**Comment by cmgiven** [comment]
Closing, and will just open a separate PR to fix the typo.

**Comment by cmgiven** [comment]
Actually, it's already been dealt with.

**Pull request closed without merging**